cases/forms/recurrence.py

```diff
@@ -75,6 +75,16 @@ class EffectForm(StepForm):
     )
 
 
+class InternalFlagsForm(StepForm):
+    submit_text = "Submit"
```

Comment:
What's wrong with the default Next - it goes from here to the summary page?

Reply:
My bad, appears to be a copy-paste artefact from the `SummaryForm` that slipped past me.